### PR TITLE
Create Desktop Version

### PR DESCRIPTION
**Pull request by kwekubright**
### Description:

In this milestone, I have achieved the following:

- Added styles to make the project responsive for desktop devices 🖥️ 
- Updated background images with high-resolution 🔆  version s
- Made navigation sticky for desktop devices

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,121 +63,140 @@ <h3>Let's connect</h3>
...
-            <a href="#">
-              <img class="social f" src="images/icons/f-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-        </ul>
-        <button type="button" class="button button-enabled">See my resume</button>
+        <div class="about-details-wrap">
+          <h2 class="section-title">About <br> myself</h2>
+          <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a href="#" class="headline-cta margin-top-12">
```
Comment (thecodechaser) [inline index.html:167]: - [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) design as much as possible. Please make sure that the skills section contains approx 60% of the total width and the left section contains the remaining and it should match with the Figma design.

### Your design

image

### Figma design

image

Comment (thecodechaser) [inline index.html:167]: - [ ] Please remove the left margin from the language icons section and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -226,7 +250,7 @@ <h4>Skills</h4>
       </div>
     </section>
 
-    <footer class="flex footer">
+    <footer id="contact" class="flex footer">
       <div class="flex footer-wrapper">
         <h2 class="section-title">Contact me</h2>
         <p class="section-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
```
Comment (thecodechaser) [inline index.html:256]: - [ ] Please reduce the gap between the contact section paragraph and the form and make sure it matches the Figma design.

### Your design

image

### Figma design

image

Comment (thecodechaser) [inline index.html:256]: - [ ] The width of the form fields is a little bigger than the design. Please reduce the form fields width and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kwekubright👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @kwekubright, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**